src/display/font_loader.js

```diff
@@ -216,6 +216,8 @@ if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('MOZCENTRAL')) {
 
       var i, ii;
 
+      // CanvasFactory is not needed here, as it is used to determine if fonts
```

Comment:
Here and below, replace "CanvasFractory is not needed here, as it is" to something like "The temporary canvas is"